src/scale.jl

```diff
@@ -1,5 +1,6 @@
 using MIDI
 using StatsBase
+using DataStructures
```

Comment:
You need to add DataStructures to Project.toml. Also need to add a compat version for it.